filters/include/pcl/filters/crop_hull.h

```diff
@@ -177,17 +169,6 @@ namespace pcl
          *                    filter.
          */
       void
-      applyFilter3D (PointCloud &output);
-
-      /** \brief Apply the three-dimensional hull filter.
-        *  Polygon-ray crossings are used for three rays cast from each point
-        *  being tested, and a  majority vote of the resulting
-        *  polygon-crossings is used to decide  whether the point lies inside
-        *  or outside the hull.
-        * \param[out] indices The indices of the set of points that pass the 3D
```

Comment:
You are deleting the wrong documentation here

Reply:
It is my fault. I was careless.
Thanks you for finding this mistake.